src/components/gameMenu.html

```diff
@@ -14,26 +14,29 @@
             <button class="dropdown-item" href="#">Stats</button>
         </li> -->
         <li>
-            <a class="dropdown-item" href="https://pokeclicker.reddit.com" target="_blank">Subreddit</a>
+            <a class="dropdown-item" href="#achievementsModal" data-toggle="modal">Achievements</a>
         </li>
         <li>
-            <a class="dropdown-item" href="#saveModal" data-toggle="modal">Save</a>
+            <a class="dropdown-item" href="#shardModal" data-toggle="modal">Shards</a>
         </li>
         <li>
-            <a class="dropdown-item" href="#achievementsModal" data-toggle="modal">Achievements</a>
+            <a class="dropdown-item" href="#showItemsModal" data-toggle="modal">Items</a>
         </li>
         <li>
-            <button class="dropdown-item" href="#changelogModal" data-toggle="modal">Changelog</button>
+            <a class="dropdown-item" href="#saveModal" data-toggle="modal">Save</a>
         </li>
         <li>
-            <a class="dropdown-item" href="#shardModal" data-toggle="modal">Shards</a>
+            <button class="dropdown-item" href="#changelogModal" data-toggle="modal">Changelog</button>
```

Comment:
This one seems to be the only `button` element, not really an issue, but may as well change it now.
```suggestion
            <a class="dropdown-item" href="#changelogModal" data-toggle="modal">Changelog</a>
```

Reply:
Huh that is weird